elastic/kibana
# [performance] use journey own ftr config to run scalability test

While debugging scalability testing failure for `cloud_security_dashboard` journey, I found that we hardcoded base FTR config to `x-pack/performance/journeys/login.ts` and main issue is that Kibana is not started properly.

This PR makes few changes:
- update `kbn-performance-testing-dataset-extractor` to save journey path as `configPath` so it can be later used to start ES/Kibana in the scalability run with the same configuration it was run for the single user journey run.
- update scalability entry configuration to read base FTR config from generated scalability json file (`configPath` property)

How to test:
- make sure to clone the latest [kibana-load-testing](https://github.com/elastic/kibana-load-testing) repo and build it `mvn clean test-compile`
- from kibana root directory run any api capacity test
```
node scripts/run_scalability.js --journey-path x-pack/test/scalability/apis/api.core.capabilities.json
```
Expected result: logs should display
```
debg Loading config file from x-pack/performance/journeys/login.ts
```
- download the latest artifacts from [buildkite](https://buildkite.com/elastic/kibana-performance-data-set-extraction/builds/171#0186a342-9dea-4a9b-bbe4-c96449563269), find `cloud_security_dashboard-<uuid>.json`
- from kibana root directory run scalability test for `cloud_security_dashboard` journey
```
node scripts/run_scalability.js --journey-path <path to cloud_security_dashboard-<uuid>.json>
```
Expected result: logs should display 
```
debg Loading config file from x-pack/performance/journeys/cloud_security_dashboard.ts
```
